Makefile

```diff
@@ -1,7 +1,8 @@
+# N - in response to Would you like to add the SemaphoreCI addon?
```

Comment:
small thing @longnd I would like to set this one as Y because, for the test, we would like to go through all addons to make sure all of that addons work, if we ignore the Semaphore CI here, this might not work as it isn't applied during the test. What do you think?

Reply:
that makes sense. Updated in 71c0d78.